src/main/java/be/uantwerpen/minelabs/renderer/BohrBlueprintEntityRenderer.java

```diff
@@ -33,6 +31,129 @@ public class BohrBlueprintEntityRenderer extends EntityRenderer<BohrBlueprintEnt
 
     // radius usable for atom rendering
     private static final float MAX_RENDER_RADIUS = 0.75f + 11f / 16f - 0.1f;
+
+    // maps the number of electrons to the correct shell configuration
+    private static final Map<Integer, int[]> ELECTRON_SHELL_MAPPINGS = new HashMap<Integer, int[]>() {{
```

Comment:
Instead of mapping from int to something it might be more efficient to use a list and index it with the integer value.

Reply:
True, it's now an array